### PR TITLE
Add Interactivity and Polling on LTS pages

### PR DESCRIPTION
**Pull request by Vandit1604**
Re-used [rate.js](https://github.com/jenkins-infra/jenkins.io/blob/9741a93d5f031e575f60d88a6bb11b56c04575da/content/files/rate.js) to add the functionality of giving feedback from the svg weather icons present in LTS pages.
I've tried seperating it as another component `Rating.js`

Please do give your feedback if I could do it in a better way.

### CODE REVIEW

File: blog/src/components/Ratings.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
+import React, { useEffect } from 'react';
+// @author Alan Harder
+
```
Comment (iamrajiv) [inline blog/src/components/Ratings.js:3]: The code currently conducts various DOM manipulations within the **`loaddata`** and **`do_loaddata`** functions. A more efficient approach would be to execute these manipulations outside the loop and then append the complete content to the DOM in one go.

To enhance efficiency, you can utilize document fragments to group DOM manipulations together.

**Review by iamrajiv — CHANGES_REQUESTED** [review]

File: blog/src/components/Ratings.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+function health(nm, cls, ver, rate, desc) {
+  return '<img src="../../images/images/changelog/' + nm + '.svg" onclick="rate(\'' + ver + '\',' + rate + ')" class="rate ' + cls + '" alt="' + nm + '" title="' + desc + '"/>';
+}
+
+function do_loaddata() {
+  var r, v, j = true, div1, div2, txt;
+  for (var anchors = document.getElementsByTagName('H3'), i = 0; i < anchors.length; i++) {
+    if (anchors[i].id.charAt(0) != 'v') continue;
+    r = data[v = anchors[i].id.substring(1)];
```
Comment (krisstern) [inline blog/src/components/Ratings.js:25]: Did this code work for you @Vandit1604? 

I am currently getting the following errors in my stack trace when I tried to run the code locally:
```
/Users/krisstern/jenkins-docs/blog/src/components/Ratings.js
  25:9   error  'data' is not defined         no-undef
  59:50  error  Unexpected use of 'location'  no-restricted-globals
  62:25  error  Unexpected use of 'location'  no-restricted-globals
```

The script is not working for me.

**Review by krisstern — CHANGES_REQUESTED** [review]

File: blog/src/components/Ratings.js (hunk 2 above)
Comment (Vandit1604) [reply blog/src/components/Ratings.js:25]: Yeah. I'm getting the same errors. I'm working on it right now.

Comment (krisstern) [reply blog/src/components/Ratings.js:26]: I added a commit to patch them for now, but it appears that the three buttons are not working on say the Changelog page. There is no reaction after I clicked on each.

**Comment by Vandit1604** [comment]
Not pushing to this branch anymore. It looks really confusing now. I'll raise another PR only for `localhost:8000/changelog-lts/` page.

**Comment by Vandit1604** [comment]
Closing this. Check #107 for Features added to Changelogs.

**Pull request closed without merging**